data/data/azure/vnet/internal-lb.tf

```diff
@@ -0,0 +1,69 @@
+resource "azurerm_lb" "internal" {
+  sku                 = "Standard"
+  name                = "${var.cluster_id}-ilb"
+  resource_group_name = "${var.resource_group_name}"
+  location            = "${var.region}"
+  
+  frontend_ip_configuration {
+    name                 = "internal"
+    subnet_id = "${azurerm_subnet.master_subnet.id}"
+  }
+}
+
+resource "azurerm_lb_backend_address_pool" "master_ilb_pool" {
+  resource_group_name = "${var.resource_group_name}"
+  loadbalancer_id = "${azurerm_lb.internal.id}"
+  name = "${var.cluster_id}-ilb-master"
+}
+
+resource "azurerm_lb_rule" "internal_lb_rule_api_internal" {
+  name = "api-internal"
+  resource_group_name = "${var.resource_group_name}"
+  protocol="Tcp"
+  backend_address_pool_id = "${azurerm_lb_backend_address_pool.master_ilb_pool.id}"
+  loadbalancer_id = "${azurerm_lb.internal.id}"
+  frontend_port = 6443
+  backend_port = 6443
+  frontend_ip_configuration_name = "internal"
+  enable_floating_ip = false
+  idle_timeout_in_minutes = 4
+  load_distribution = "Default"
+  probe_id = "${azurerm_lb_probe.internal_lb_probe_api_internal.id}"
+}
+
+resource "azurerm_lb_rule" "internal_lb_rule_sint" {
```

Comment:
Whats this?

Reply:
This is the endpoint that the masters query to get their setup from the bootstrap machine, during ignition.
sint stands for service_internal I think. taken from their aws implementation.
Their is a corresponding probe, and only the bootstrap machine will satisfy it, so the call is effectively only routed to the bootstrap machine.